cloud/customize-configuration.md

```diff
@@ -62,6 +62,20 @@ As with the basic database configuration parameters, any changes are highlighted
 and the `Apply changes` (or `Restart and apply changes`) button is available to
 click, prompting you to confirm any changes before the Service is modified.
 
+## Multiple databases
+To create more than one Timescale Cloud database, you need to create a new 
+service for each database. Timescale Cloud does not support multiple 
+databases within the same service. Having a separate service for each database 
+affords each database its own isolated resources. 
+
+Another option is to use [schemas](https://www.postgresql.org/docs/current/ddl-schemas.html). Schemas provide a way to organize 
+tables into logical groups. A single database can contain multiple schemas, 
+which in turn contain tables. The main difference between isolating with 
+databases versus schemas is that a user can access objects in any of the 
+schemas in the database they are connected to, so long as they have the 
+corresponding privileges. Schemas can help isolate smaller use cases 
+that do not warrant their own service.
```

Comment:
```suggestion
Another option is to use 
[schemas](https://www.postgresql.org/docs/current/ddl-schemas.html). 
Schemas provide a way to organize tables into logical groups. A single 
database can contain multiple schemas, which in turn contain tables. 
The main difference between isolating with databases versus schemas 
is that a user can access objects in any of the schemas in the database 
they are connected to, so long as they have the corresponding privileges. 
Schemas can help isolate smaller use cases that do not warrant their 
own service.
```

Reply:
Instead of all this info about what a schema is, why not just link out to our own Schema how to? (I note you've linked to the pg docs, not internally). Then, you don't need to explain what schemas are, necessarily, you can just say something like "if you want to run multiple datasets within a single service, you can use schemas within a single database. For more information about schemas, see the [Schema documentation](https://docs.timescale.com/timescaledb/latest/how-to-guides/schema-management/)" (or similar).